### ISSUE TITLE
[Android] Double pairing modal

### ISSUE THREAD

**Issue by Tadejo9** [dotintent/react-native-ble-plx#877]
## Question
I am connecting to weight scale on iOS and pair modal pops up only once. As it should. But on Android, bonding is triggered on discovery and then again on reading protected characteristic.  I console logged App state.

````
Connecting to deviceName with id deviceId.
Discovering deviceName with id deviceId.
APP STATE:  background
Reading characteristic to pair with deviceName with id deviceId.
APP STATE:  active
APP STATE:  background
APP STATE:  active
```

App state background corresponds to modal becoming visible and active to modal hiding.

Is this a bug with library or it depends on Android version/device.

Device model: SM-j415fn
Android version: 9

**Comment by abursztynowska** [comment]
I'm excited to inform you that we've released a new version of `react-native-ble-plx`. Many of the concerns and bugs have been addressed in this update.

If you encounter any issues or have further suggestions, please don't hesitate to open a new issue.
